app/views/videos/_reviewform.html.haml

```diff
@@ -0,0 +1,8 @@
+= bootstrap_form_for([@video,@review]) do |f|
+  .row
+    .col-sm-3
+      =f.select :rating, [["Select",nil], ["1 Star",1.0], ["2 Star", 2.0], ["3 Star", 3.0], ["4 Star", 4.0], ["5 Star", 5.0]], {label: "Rate this video"}
```

Comment:
You should not hardcode options. Calculate them and re-use the helper method which you already built.